frontend/express/public/javascripts/dom/tipsy/jquery.tipsy.js

```diff
@@ -23,8 +23,26 @@
                 var $tip = this.tip();
                 
                 $tip.find('.tipsy-inner')[this.options.html ? 'html' : 'text'](title);
-                $tip[0].className = 'tipsy '+this.options.cssClass; // reset classname in case of dynamic gravity
```

Comment:
Did you recheck other places that used tipsy and if they are not broken now? 
I searched repo and some use cssClass option

Reply:
ok, let me resume the customized class name to tipsy view.